internal/agent/agent.go

```diff
@@ -204,7 +204,9 @@ func New(conf *Config) (*Agent, error) {
 		newLabs:    make(chan pb.Lab, 10),
 		State: &State{
 			ExClient: exClient,
-			EnvPool:  &env.EnvPool{},
+			EnvPool: &env.EnvPool{
+				Envs: map[string]env.Environment{},
```

Comment:
```suggestion
				Envs: map[string]env.Environment{},
				Envs: make(map[string]env.Environment),
```
I think best practice would be to use make to initialize maps

Reply:
Will take into account!